wbooks/src/redux/book/actions.ts

```diff
@@ -12,11 +13,11 @@ const actionCreators = {
     target: 'books',
     service: getBooks
   }),
-  getFilteredBooks: (query: string) => ({
+  getFilteredBooks: (data: FilteredBooks) => ({
     type: actions.GET_FILTERED_BOOKS,
     target: 'filteredBooks',
     service: getFilteredBooks,
-    payload: query
+    payload: data
```

Comment:
No, you must save the `query` and use the hook/filter in the `Filter` screen only

Reply:
Done!